packages/clay-popover/src/index.tsx

```diff
@@ -187,6 +187,20 @@ const ClayPopover = React.forwardRef<HTMLDivElement, IProps>(
 			}
 		}, [closeOnClickOutside, trigger]);
 
+		useEffect(() => {
+			const handleKeyDown = (event: KeyboardEvent) => {
+				if (event.key === 'Escape') {
```

Comment:
Hey @julien, when reading https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key, I notice we should use `Esc` for representing `Escape` key to make it work with IE/Edge too. 

Nevermind, [it normalizes the key](https://github.com/facebook/react/blob/cae635054e17a6f107a39d328649137b83f25972/packages/react-dom/src/events/SyntheticEvent.js#L300)

Reply:
:+1: and additionally Edge is Chrome with some fancy UI stuff on top. As for IE, I personally think it's time we stop supporting it - we're in 2022 after all.

Reply:
> we're in 2022 after all.

![future](https://media.giphy.com/media/fH985LNdqFZXOFHygK/giphy.gif)